src/components/Collection/Episode/EpisodeDetails.tsx

```diff
@@ -13,6 +13,10 @@ const getDuration = (duration: string) => {
   return `${intMinutes} minutes`;
 };
 
+const handleCopyToClipboard = (id: string) => {
+  copyToClipboard(id, 'Shoko episodeID').catch(console.error);
```

Comment:
maybe name it `Shoko Episode ID`

Reply:
That does look better in the toast (changed the icon of the button too)